libs/auth/src/inserted_smart_card_auth.ts

```diff
@@ -205,7 +207,7 @@ export class InsertedSmartCardAuth implements InsertedSmartCardAuthApi {
       return {
         status: 'logged_in',
         user: this.cardlessVoterUser,
-        sessionExpiresAt: computeSessionEndTime(machineState),
+        sessionExpiresAt: NEVER,
```

Comment:
Maybe leave a comment explaining why it's safe to use `NEVER` for this field?

Reply:
Ah, good call - will do